folly/detail/RangeSse42.cpp

```diff
@@ -15,9 +15,7 @@
  */
 
 #include <folly/detail/RangeSse42.h>
-
```

Comment:
Please don't remove the blank lines, they are needed to keep clang-format from re-arranging the grouping.

Reply:
Fixed.